TransIT.BLL/Helpers/MimeType.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+
+namespace TransIT.BLL.Helpers
+{
+    public class MimeType
+    {
+        private static readonly byte[] PDF = { 37, 80, 68, 70, 45, 49, 46 };
```

Comment:
do not use magic numbers anti-pattern, create constants for each number with appropriate naming and meaning, I don't understand what are these numbers, check all such cases in your pull request

Reply:
These numbers have just one meaning, value that PDF file starts with. I left a comment for better understanding.